api/recipient/recipientModel.js

```diff
@@ -1,6 +1,6 @@
 const knex = require('../../data/db-config');
 
-const findAll = () => {
+const findAll = (filter = {}) => {
```

Comment:
This is cool! Great idea to add a filter and use a default value if nothing is passed in!